src/CloudNative.CloudEvents.Kafka/KafkaExtensions.cs

```diff
@@ -32,7 +33,7 @@ public static class KafkaExtensions
         /// </remarks>
         /// <param name="message">The message to check for the presence of a CloudEvent. Must not be null.</param>
         /// <returns>true, if the request is a CloudEvent</returns>
-        public static bool IsCloudEvent(this Message<string?, byte[]> message) =>
+        public static bool IsCloudEvent<TKey>(this Message<TKey, byte[]> message) =>
```

Comment:
Note: this is a binary breaking change. (We'll need to add a generic overload to the existing non-generic method.)

Reply:
Added overload for binary compatibility